persist/fs/index_lookup_prop_test.go

```diff
@@ -0,0 +1,236 @@
+// +build big
```

Comment:
Can't run this normally? =[

Reply:
Test takes about 20 seconds to run and if I run it normally it'll timeout.

I could reduce the number of tests but then its not much of a prop test. To be clear, with the "big" build tag it will still be run in C.I, it's just run serially with all the other big tests at the end

Reply:
Ah yes, forgot about the fact that we run these at the end.  The `adhoc` tag was the one I was thinking about.